manifests/init.pp

```diff
@@ -27,6 +27,26 @@
 #
 # === Advanced parameters:
 #
+# $enable_candlepin::   Should candlepin be installed
+#                       type:Boolean
+#                       Default value: 'true'.
```

Comment:
You can see the default value in params.pp.  The rest of the params don't have this, and the problem is they become stale really fast so I'd prefer if we didn't have them.